backup-and-restore-using-dumpling-lightning.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Use Dumpling and TiDB Lightning for Data Backup and Restoration
+summary: Introduce how to use Dumpling and TiDB Lightning to backup and restore full data of TiDB.
+aliases: ['/docs-cn/dev/export-or-backup-using-dumpling/','/zh/tidb/dev/export-or-backup-using-dumpling']
```

Comment:
Fixed the two aliases in https://github.com/pingcap/docs/commit/a0ec32a0baf8ea902457089ce0c357fe8c1e4fa2.